contracts/TracerPerpetualSwaps.sol

```diff
@@ -453,13 +453,10 @@ contract TracerPerpetualSwaps is
             Balances.minimumMargin(pos, price, gasCost, maxLeverage);
         int256 margin = Balances.margin(pos, price);
 
-        if (margin < 0) {
-            /* Margin being less than 0 is always invalid, even if position is 0.
-               This could happen if user attempts to over-withdraw */
-            return false;
-        }
```

Comment:
Shouldn't this still be in anyway?

Reply:
@BenjaminPatch `Balances.minimumMargin` now returns a uint so it cant be less than 0. I think this existed as previously we returned a int and then casted so we needed to make this safety check.

Reply:
This is margin though, not minimum margin